project/ScalaOptionParser.scala

```diff
@@ -106,7 +106,7 @@ object ScalaOptionParser {
     "-Ymacro-expand" -> List("discard", "none"),
     "-Yresolve-term-conflict" -> List("error", "object", "package"),
     "-g" -> List("line", "none", "notailcails", "source", "vars"),
-    "-target" -> List("jvm-1.5", "jvm-1.6", "jvm-1.7", "jvm-1.8"))
+    "-target" -> List("jvm-1.8", "jvm-9", "9"))
```

Comment:
Needs to go up to 12 as well.

It would be great if we could resurrect https://github.com/scala/scala-tool-support/blob/master/bash-completion/src/main/scala/BashCompletion.scala to dynamically generate the completion data for Bash and for `sbt scalac ...` as well.

Reply:
Just remembered we now have `sbt generateDocsData` which generates `compiler-options.yml` containing all the data we would need. So the "it would be great" thing would now to be that data into the our parser for `sbt scalac` _and_ to transform that data into a bash completion script.